### ISSUE TITLE
User firstName/lastName never update

### ISSUE THREAD

**Issue by kallisti5** [looorent/keycloak-admin-ruby#27]
Keycloak 21.x

Code:
```ruby
    update_name = Hash.new                                                                          
    update_name.merge!({firstName: firstname}) if firstname && firstname.length > 0                 
    update_name.merge!({lastName: lastname}) if lastname && lastname.length > 0                     
                                                                                                    
    begin                                                                                           
      user = KeycloakAdmin.realm("haiku").users.search(email).first                                 
      puts("found #{user.id} to update with...")                                                    
      pp update_name                                                                                
      pp KeycloakAdmin.realm("haiku").users.update(user.id, update_name)                            
    rescue RuntimeError => error                                                                    
        puts "FAIL - Unable to add #{username} - #{error}"                                          
        next                                                                                        
    end                                                                                             
```

Result:
```
found facd73a6-f66f-45bb-b274-XXXXXXXXXX to update with...
{:firstName=>"waddlesplash", :lastName=>"waddlesplash"}
true
```

**Comment by kallisti5** [comment]
![users](https://github.com/looorent/keycloak-admin-ruby/assets/100110/64932513-08e8-45ff-9eb1-6ee1a3c296df)

**Comment by looorent** [comment]
Thanks for your message, @kallisti5 !
It is now fixed in version `1.0.24`
